src/stacked_borrows.rs

```diff
@@ -304,26 +307,29 @@ impl<'tcx> Stack {
     /// is met: We cannot push `Uniq` onto frozen stacks.
     /// `kind` indicates which kind of reference is being created.
     fn create(&mut self, bor: Borrow, kind: RefKind) {
-        // When creating a frozen reference, freeze.  This ensures F1.
-        // We also do *not* push anything else to the stack, making sure that no nother kind
+        // When creating a frozen reference, freeze. This ensures F1.
+        // We also do **not** push anything else to the stack, making sure that no nother kind
         // of access (like writing through raw pointers) is permitted.
         if kind == RefKind::Frozen {
             let bor_t = match bor {
                 Borrow::Alias(Some(t)) => t,
                 _ => bug!("Creating illegal borrow {:?} for frozen ref", bor),
             };
-            // It is possible that we already are frozen (e.g. if we just pushed a barrier,
+            // It is possible that we already are frozen (e.g., if we just pushed a barrier,
```

Comment:
murdercommas

Reply:
I know some don't like this, but this is standard. People agreed to accept the U.S. English convention I believe. :-)

Reply:
I think what makes it look weird is the monospace font. I just looked up a few examples in arial fonts, and there it actually looks ok. But with monospace something feels just very off.

Reply:
Okay, fair enough. I'll undo these. Are you happy with this style being at odds with what's used throughout the rustc codebase though? We have to go with the convention there. (I prefer it anyway, but yeah, I'm weird.)

Reply:
Yea there's no need to keep the style in sync with rustc. We've always been at odds with the rustc style (and ended up being closer to the style guide than rustc when that styleguid was introduced)

Reply:
I am actually surprised I didn't add a comma here, usually I do and I wrote this code.^^